# Only two elements are allowed in the array. e.g. ['x0', '0']
string[] atomic_proposition 